network/v1/001-crd.yaml

```diff
@@ -0,0 +1,409 @@
+            the overlay network for its pods
+          format: int32
+          type: integer
+          minimum: 2
+          maximum: 30
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        mtu:
+          description: MTU is the MTU for the overlay network. This should be 50 less
+            than the MTU of the network connecting the nodes. It is normally autodetected
+            by the cluster network operator.
+          format: int32
+          type: integer
+          minimum: 576
+          maximum: 65536
+        network:
+          description: Network is a CIDR string specifying the global overlay network's
+            L3 space
+          type: string
+          pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$'
+        pluginName:
+          description: PluginName is the name of the network plugin being used
+          type: string
+        serviceNetwork:
+          description: ServiceNetwork is the CIDR range that Service IP addresses
+            are allocated from
+          type: string
+          pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$'
+        vxlanPort:
+          description: VXLANPort sets the VXLAN destination port used by the cluster.
+            It is set by the master configuration file on startup and cannot be edited
+            manually. Valid values for VXLANPort are integers 1-65535 inclusive and
+            if unset defaults to 4789. Changing VXLANPort allows users to resolve
+            issues between openshift SDN and other software trying to use the same
+            VXLAN destination port.
+          format: int32
+          type: integer
+          minimum: 1
+          maximum: 65535
+      required:
+      - clusterNetworks
+      - serviceNetwork
+      type: object
+  additionalPrinterColumns:
+  - name: Cluster Network
+    type: string
+    description: The primary cluster network CIDR
+    JSONPath: .network
+  - name: Service Network
+    type: string
+    description: The service network CIDR
+    JSONPath: .serviceNetwork
+  - name: Plugin Name
+    type: string
+    description: The OpenShift SDN network plug-in in use
+    JSONPath: .pluginName
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Create separate file for each type, please.

Reply:
Will do. Although, am not sure if we can consume them individually from the bindata dir
@dcbw can you confirm that splitting the crds into individual files shouldn't be an issue?

Reply:
Also, @soltysh is there a naming convention to name these files?

Reply:
There are no strong requirements on that.